experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -200,7 +242,9 @@ def addBoundingBoxesIntoAtomspace(self, features):
             imageFeatures = FloatValue(boundingBoxFeatures)
             boundingBoxInstance = ConceptNode(
                 'BoundingBox-' + str(boundingBoxNumber))
-            InheritanceLink(boundingBoxInstance, ConceptNode('BoundingBox'))
+            inh = InheritanceLink(boundingBoxInstance, ConceptNode('BoundingBox'))
+            tv = TruthValue(0.999, 0.999)
```

Comment:
Why 0.999? :-)

Reply:
I am quite sure that code above produces InheritanceLink, but not 100% sure - there could be bug or something. I believe that probabilities of zero and one are applicable only to logical statements.

Reply:
I'll try 1.0 0.999. Default stv(1, 0) gives stv(1, 0) for queries to URE with fuzzy logic formulas with AndLinks